packages/oas/src/index.ts

```diff
@@ -484,7 +482,7 @@ export default class Oas {
    * @param url A URL to swap variables into.
    * @param variables An object containing variables to swap into the URL.
    */
-  replaceUrl(url: string, variables: Variables = {}) {
+  replaceUrl(url: string, variables: RMOAS.Variables = {}) {
```

Comment:
could use `ServerVariable` instead?

```suggestion
  replaceUrl(url: string, variables: RMOAS.ServerVariable = {}) {
```

Reply:
huh didn't even notice that! ty